src/Microsoft.Extensions.Globalization.CultureInfoCache/CultureInfoCache.cs

```diff
@@ -12,6 +12,7 @@ namespace Microsoft.Extensions.Globalization
     /// <summary>
     /// Provides read-only cached instances of <see cref="CultureInfo"/>.
     /// </summary>
+    [Obsolete("This type is obsolete and will be removed in a future version.")]
     public static class CultureInfoCache
```

Comment:
Do we really need this class anymore? or is it there for backward compatibility?